### PR TITLE
Rpg miles

### PR DESCRIPTION
**Pull request by MJiang18**
Pls pull

### CODE REVIEW

File: core/src/com/mygdx/insanerpg/item/Item.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.mygdx.insanerpg.item;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/Item.java]: Look at allens solution, should be under models.

File: core/src/com/mygdx/insanerpg/item/Item.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+public class Item {
+    private int _sellPrice;
+    private int _buyPrice;
+    private String _name;
+    private String _rarity;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/Item.java]: Not sure if it should be string, look up enums.

File: core/src/com/mygdx/insanerpg/item/Item.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    private int _itemID;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/Item.java]: ID should be Id.

File: core/src/com/mygdx/insanerpg/item/equipment/Equipment.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.mygdx.insanerpg.item.equipment;
+
+import com.mygdx.insanerpg.item.Item;
+
+public class Equipment extends Item {
+    private int _durability;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/equipment/Equipment.java]: Current durability or max durability?

File: core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package com.mygdx.insanerpg.item.equipment.weapon;
+
+import com.mygdx.insanerpg.item.equipment.Equipment;
+
+public class Weapon extends Equipment {
+    private int _damage;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java]: Should damage be int? What is damage?

File: core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    private int _attackspeed;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java]: AttackSpeed might want to be enum.

File: core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    private int _range;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/item/equipment/weapon/Weapon.java]: Range of what? Is it radius or etc.

**Review by Chrischeme — CHANGES_REQUESTED** [review]

File: core/src/com/mygdx/insanerpg/item/Item.java (hunk 1 above)
Comment (MJiang18) [reply core/src/com/mygdx/insanerpg/item/Item.java]: added model package to envelope items package

File: core/src/com/mygdx/insanerpg/item/Item.java (hunk 2 above)
Comment (MJiang18) [reply core/src/com/mygdx/insanerpg/item/Item.java]: To do

File: core/src/com/mygdx/insanerpg/item/Item.java (hunk 3 above)
Comment (MJiang18) [reply core/src/com/mygdx/insanerpg/item/Item.java]: ok

File: core/src/com/mygdx/insanerpg/item/equipment/Equipment.java (hunk 4 above)
Comment (MJiang18) [reply core/src/com/mygdx/insanerpg/item/equipment/Equipment.java]: added two fields

File: core/src/com/mygdx/insanerpg/model/item/Item.java [hunk 8]
```diff
@@ -0,0 +1,8 @@
+package com.mygdx.insanerpg.model.item;
+
+public class Item {
+    private int _sellPrice;
+    private int _buyPrice;
+    private int _itemId;
+}
+
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/Item.java]: delete space

File: core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Sword.java [hunk 9]
```diff
@@ -0,0 +1,5 @@
+package com.mygdx.insanerpg.model.item.equipment.weapon;
+
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Sword.java:2]: space

File: core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Scythe.java [hunk 10]
```diff
@@ -0,0 +1,4 @@
+package com.mygdx.insanerpg.model.item.equipment.weapon;
+
+public class Scythe extends Weapon{
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Scythe.java]: space between Weapon and {

File: core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Weapon.java [hunk 11]
```diff
@@ -0,0 +1,10 @@
+package com.mygdx.insanerpg.model.item.equipment.weapon;
+
+import com.mygdx.insanerpg.model.item.equipment.Equipment;
+
+public class Weapon extends Equipment {
+    private int _basePhysicalDamage;
+    private int _baseMagicalDamage;
+    private boolean _isOneHanded;
+
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/equipment/weapon/Weapon.java]: space

File: core/src/com/mygdx/insanerpg/model/item/stackable/consumable/Potion.java [hunk 12]
```diff
@@ -0,0 +1,5 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable;
+
+public class Potion extends Consumable {
+
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/stackable/consumable/Potion.java]: space

File: core/src/com/mygdx/insanerpg/model/item/stackable/consumable/Consumable.java [hunk 13]
```diff
@@ -0,0 +1,10 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable;
+
+import com.mygdx.insanerpg.model.item.Item;
+
+public class Consumable extends Item {
+    private int _cooldown;
+    private int _hasDelay;
```
Comment (Chrischeme) [inline core/src/com/mygdx/insanerpg/model/item/stackable/consumable/Consumable.java]: buddy this is not an int

**Review by Chrischeme — CHANGES_REQUESTED** [review]

**Review by Chrischeme — APPROVED** [review]
Sure why not.

**Pull request merged**